ui/components/multichain/app-header/app-header.test.js

```diff
@@ -175,4 +185,48 @@ describe('App Header', () => {
       expect(connectionPickerButton).not.toBeInTheDocument();
     });
   });
+
+  describe('network picker', () => {
+    it('shows custom rpc if it has the same chainId as a default network', () => {
+      const mockProviderConfig = {
+        id: 'custom-rpc-localhost',
+        type: 'rpc',
+        ticker: 'ETH',
+        chainId: '0x1',
+        rpcUrl: 'https://localhost:8545',
+        nickname: 'Localhost',
+      };
```

Comment:
Also, this one is being used twice, maybe we could move it to the top-level? We could also rename it `mockCustomProviderConfig` WDYT?

Reply:
There is a small difference between the two providers configs

Reply:
that's my bad then 🙈